for_time.py

```diff
@@ -0,0 +1,460 @@
+import time
+import datetime
+import os
+
+while True:
+    dn = datetime.datetime.now().time()
+    str = dn.strftime("%H:%M:%S")
+    list = str.split(":")
+    hours, minutes, seconds = list
+    hour_0 = int(hours[0])
+    hour_1 = int(hours[1])
+    minute_0 = int(minutes[0])
+    minute_1 = int(minutes[1])
+    second_0 = int(seconds[0])
+    second_1 = int(seconds[1])
+
+    
+    if hour_0 == 0:
```

Comment:
I would create a separate structure for  entire digit let say
[
[x,x,x,x],
[x,x,x,x]
]
like this
and put all the digits into a dictionary to get rid of all the IFs